media-video/mjpg-streamer/mjpg-streamer-0_pre20250324-r1.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
```

Comment:
Why is this not using `cmake.eclass` in the first place? All my review comments are to be read in that context.